flexmeasures/ui/views/charts.py

```diff
@@ -0,0 +1,129 @@
+from flask_security import roles_accepted
+from bokeh.embed import json_item
+from marshmallow import Schema, fields
+from webargs.flaskparser import use_args
+from flask_cors import cross_origin
+from flask_json import as_json
+
+from flexmeasures.api.v2_0 import flexmeasures_api as flexmeasures_api_v2_0
+from flexmeasures.api.v2_0.routes import v2_0_service_listing
+from flexmeasures.api.common.schemas import PeriodField
+from flexmeasures.data.queries.analytics import get_power_data
+from flexmeasures.ui.views.analytics import make_power_figure
+
+
+"""
+An endpoint to get a power chart.
+
+This will grow to become code for more charts eventually.
+The plan is to separate charts specs from the actual data later,
+and to switch to Altair.
+
+For now, we'll keep this endpoint here, with route and implementation in the same file.
+When we move forward, we'll review the architecture.
+"""
+
+
+v2_0_service_listing["services"].append(
+    {
+        "name": "POST /charts/power",
```

Comment:
GET, or am I confused? The endpoint name suggests a GET endpoint.

Reply:
I was undecided, but as the PR description says (have you read it?) it is a TODO to get the parameters as form data.

Reply:
Got it. I had missed that comment.